src/components/dropdown-filter-ajax/__spec__.js

```diff
@@ -291,6 +291,30 @@ describe('DropdownFilterAjax', () => {
         });
       });
     });
+
+    describe('when the param withCredentials is passed', () => {
+      beforeEach(() => {
+        instance = TestUtils.renderIntoDocument(
+          <DropdownFilterAjax
+            name="foo"
+            value="1"
+            path="/foobar"
+            create={ function() {} }
+            withCredentials
+            additionalRequestParams={ {foo: 'bar'} }
```

Comment:
Spaces inside an object (linting didn't get this?)

`{ foo: 'bar' }`

Reply:
Specs appear to be added to the eslint ignore.

Reply:
Carbon hasn't caught up yet then :-P

I'd still change it for consistency, but it's not a blocker!